python/lsst/meas/base/apCorrRegistry.py

```diff
@@ -20,29 +20,29 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-"""Registry for names of flux fields that should be aperture corrected
+"""Registry for names of instFlux fields that should be aperture corrected
 """
 
 __all__ = ("addApCorrName", "getApCorrNameSet")
 
-# Set of names of algorithms that measure fluxes that can be aperture corrected
+# Set of names of algorithms that measure instFluxes that can be aperture corrected
```

Comment:
Consider `instrument fluxes` instead of `instFluxes` here, and if you change t please also look for other instances and consider if they would also be clearer spelled out.